src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button } from "@chakra-ui/react"
+import { Story, ComponentMeta } from "@storybook/react"
+
+import { Footer } from "./Footer"
+
+const footerMeta = {
+  title: "Components/Footer",
+  component: Footer,
+} as ComponentMeta<typeof Footer>
+
+interface TemplateArgs {
+  buttonText: string
+}
+
+const SingleButtonFooterTemplate: Story<TemplateArgs> = ({
+  buttonText,
+}: TemplateArgs) => {
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
nit: extra comment

Reply:
Thanks for catching, https://github.com/isomerpages/isomercms-frontend/pull/932/commits/701f438ffdf74c5dc631deb36a01f621f694d103